doc/mesh_filter/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+add_executable(filtered_depth_subscriber_node src/filtered_depth_subscriber.cpp)
+target_link_libraries(filtered_depth_subscriber_node ${catkin_LIBRARIES} ${Boost_LIBRARIES})
+install(TARGETS filtered_depth_subscriber_node DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION})
+
+install(DIRECTORY launch DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION})
```

Comment:
you need to install the urdfs/meshes as well (and make sure the tutorial finds them in an install workspace).